JiraPS/Public/New-JiraIssue.ps1

```diff
@@ -85,6 +95,10 @@ function New-JiraIssue {
         if ($PSCmdlet.MyInvocation.BoundParameters.ContainsKey("Reporter")) {
             $requestBody["reporter"] = @{"name" = "$Reporter"}
         }
+        elseif ($ProjectObj.Style -eq "next-gen"){
+            Write-DebugMessage "[$($MyInvocation.MyCommand.Name)] Adding reporter as next-gen projects must have reporter set."
+            $requestBody["reporter"] = @{"name" = "$((Get-JiraUser -UserName ((Get-JiraSession).UserName)).Name)"}
```

Comment:
You can user `Get-JiraUser` without parameters. This way, it will also work if the call is being executed without using `New-JiraSession`.
(Of course, you need to pass `-Credentials` on)

https://github.com/AtlassianPS/JiraPS/blob/a0e29a616fda283531d8d127dc031dc2efd9b5fd/JiraPS/Public/Get-JiraUser.ps1#L64-L76

Reply:
Added a new commit with this change.